docs/collecting-application-metrics.md

```diff
@@ -0,0 +1,211 @@
+# Collecting application metrics
+
+This document covers multiple different use cases related to scraping custom application metrics.
```

Comment:
```suggestion
This document covers multiple different use cases related to scraping custom application metrics exposed in Prometheus format.
```

Reply:
It's not exactly true, but I will make few more iterations on that doc, so we can improve it later